src/plugins/vis_type_timelion/public/helpers/timelion_request_handler.ts

```diff
@@ -93,7 +96,13 @@ export function getTimelionRequestHandler({
 
     // parse the time range client side to make sure it behaves like other charts
     const timeRangeBounds = timefilter.calculateBounds(timeRange);
-    const sessionId = getDataSearch().session.getSessionId();
+    const untrackSearch =
```

Comment:
I don't see why this needs to be done at all.
The server side search service is responsible for tracking searches.

Reply:
this UI side only to track the current state of the search session: `loading / completed / backgroundLoading / backgroundCompleted`